src/js/components/Menu/Menu.js

```diff
@@ -288,6 +300,92 @@ const Menu = forwardRef((props, ref) => {
     </Box>
   );
 
+  const menuItem = (item, index) => {
+    // Determine whether the label is done as a child or
+    // as an option Button kind property.
+    const child = !theme.button.option ? (
+      <Box
+        align="start"
+        pad="small"
+        direction="row"
+        gap={item.gap}
+        justify={item.justify}
+      >
+        {item.reverse && item.label}
+        {item.icon}
+        {!item.reverse && item.label}
+      </Box>
+    ) : undefined;
+
+    // if we have a child, turn on plain, and hoverIndicator
+    return (
+      // eslint-disable-next-line react/no-array-index-key
+      <Box key={index} flex={false} role="none">
+        <Button
+          ref={(r) => {
+            buttonRefs.current[index] = r;
+          }}
+          role="menuitem"
+          onFocus={() => {
+            setActiveItemIndex(index);
+          }}
+          active={activeItemIndex === index}
+          focusIndicator={false}
+          plain={!child ? undefined : true}
+          align="start"
+          justify={item.justify}
+          kind={!child ? 'option' : undefined}
+          hoverIndicator={!child ? undefined : 'background'}
+          {...(!child
+            ? item
+            : {
+                ...item,
+                gap: undefined,
+                icon: undefined,
+                label: undefined,
+                reverse: undefined,
+              })}
+          onClick={(...args) => {
+            if (item.onClick) {
+              item.onClick(...args);
+            }
+            if (item.close !== false) {
+              onDropClose();
+            }
+          }}
+        >
+          {child}
+        </Button>
+      </Box>
+    );
+  };
+
+  let menuContent;
+  if (itemCount && Array.isArray(items[0])) {
+    menuContent = items.map((group, groupIndex) => (
+      <Box
+        // eslint-disable-next-line react/no-array-index-key
+        key={groupIndex}
+        {...theme.menu.group?.container}
+        border={
+          groupIndex > 0
+            ? { side: 'top', ...theme.menu.group?.container?.border }
```

Comment:
See my comments on base.js theme related to this

Reply:
Done.